playbooks/debug_box.yml

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
I'm thinking the playbook name could be a bit more descriptive. This is also doing two things that feel disjoint. Running a sos_report and fetching bats results. Maybe `gather_debug` ? `debug` ? or `collect_debug` ?